workitem/link/link_repository_blackbox_test.go

```diff
@@ -123,6 +121,17 @@ func (s *linkRepoBlackBoxTest) TestDisallowMultipleParents() {
 	require.Nil(s.T(), err)
 	s.testTreeLinkTypeID = TestTreeLinkType.ID
 
+	// Create link type combination
+	combiRepo := link.NewWorkItemLinkTypeCombinationRepository(s.DB)
+	combi := link.WorkItemLinkTypeCombination{
+		LinkTypeID:   TestTreeLinkType.ID,
```

Comment:
Perhaps if we have a test where the source and target wit are different could be interesting.